docs/environmentvariables.md

```diff
@@ -0,0 +1,15 @@
+
+# Environment variables
+
+
+| Variable | Default value |
```

Comment:
there are many more env vars than this, like API_URL, NODE_ENV, DISQUS_SITE, BASE_URL